app.txt

```diff
@@ -0,0 +1,4 @@
+app id:599677783926955
+app secret:67cadcef7979641022580aa20af64338
+accesstoken:EAAIhZA11KaKsBAP3DmZAq9HmrwSNmZATmZCjVqY5QNdUcM7YKt2UVWZBsAyhYTm3ME364WUmSutiNVOWFy1AdDd1qIelxbp1yfXqZBmEgkTMFJEKPnr9MZAZBnOvyWZBZB3bNsVegWODKQ3IOjRlhXOlXu1I5WuZBhmFhB9hgZBmGBPyIWMttPwGS8AT71jkUyzlHdJw9Qp3I2TlrgZDZD
```

Comment:
Do not push it to github .Add it to `.gitignore`. It has been compromised now, you will have to change it .The correct way was to add `app.txt` to `.gitignore` and mention how to make their own app.txt in the README